nuopc.runconfig

```diff
@@ -286,7 +286,7 @@ CLOCK_attributes::
      history_ymd = -999
      ice_cpl_dt = 99999 #not used
      lnd_cpl_dt = 99999 #not used
-     ocn_cpl_dt =  3600
+     ocn_cpl_dt =  1800
```

Comment:
It is not necessarily to change the value of `ocn_cpl_dt` unless `stop_option` is `nsteps`. I can do a cherry pick to add comments right after this parameter for all configurations. https://github.com/COSIMA/access-om3/issues/157#issuecomment-2249623984